docs/src/development/local/_index.md

```diff
@@ -13,12 +13,14 @@ you need to set up a development environment on your computer.
 
 Make sure that:
 
+
 - You have signed up for a Platform.sh account. 
   As a new user, you can sign up for a [free trial account](https://auth.api.platform.sh/register). 
 - You have started a [template project](../../development/templates.md) 
   or pushed your own code to Platform.sh.
+- You have [installed Git](https://docs.github.com/en/get-started/quickstart/set-up-git).
+- You have [installed the Platform.sh CLI](../../administration/cli/_index.md).
 - Optional but recommended: 
```

Comment:
:hammer_and_wrench:
Since this is now covered in the last step, it might be superfluous.

Reply:
I did consider it, but I think I'd rather keep the introduction to DDEV-like tools right at the top so that users know about them from the start/switch to them straightaway if they want to (especially since DDEV seems to be our most recommended option). Likewise, I would keep the intro to tethered/untethered there for the sake of consistency, and so users know about all the options before actually going through the different steps.